.github/workflows/elixir.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     defaults:
       run:
         working-directory: ./elixir
+    permissions:
+      checks: write
```

Comment:
Will it also grant other permissions for not whitelisted contributors?

Reply:
No, checks won't run for forks of our repo, so we should be good there.

Reply:
This is the same fix I applied in fa54a03b76 -- just needed here now too.